arch/csr/mie.yaml

```diff
@@ -5,147 +5,8 @@ mie:
-    the interrupt-pending and interrupt-enable bits for supervisor-level
-    timer interrupts. STIP is writable in `mip`, and may be written by
-    M-mode software to deliver timer interrupts to S-mode.
-
-    If supervisor mode is implemented, bits `mip`.SSIP and `mie`.SSIE are
-    the interrupt-pending and interrupt-enable bits for supervisor-level
-    software interrupts. SSIP is writable in `mip` and may also be set to 1
-    by a platform-specific interrupt controller.
-
-    <%- if ext?(:Sscofpmf) -%>
-    Bits `mip`.LCOFIP and `mie`.LCOFIE
-    are the interrupt-pending and interrupt-enable bits for local counter-overflow
-    interrupts.
-    LCOFIP is read-write in `mip` and reflects the occurrence of a local
-    counter-overflow overflow interrupt request resulting from any of the
-    `mhpmevent__n__`.OF bits being set.
-    If the Sscofpmf extension is not implemented, `mip`.LCOFIP and `mie`.LCOFIE are
-    read-only zeros.
-    <%- end -%>
-
-    Multiple simultaneous interrupts destined for M-mode are handled in the
-    following decreasing priority order: MEI, MSI, MTI, SEI, SSI, STI, LCOFI.
-
-    [NOTE]
-    ====
-    The machine-level interrupt fixed-priority ordering rules were developed
-    with the following rationale.
-
-    Interrupts for higher privilege modes must be serviced before interrupts
-    for lower privilege modes to support preemption.
-
-    The platform-specific machine-level interrupt sources in bits 16 and
-    above have platform-specific priority, but are typically chosen to have
-    the highest service priority to support very fast local vectored
-    interrupts.
-
-    External interrupts are handled before internal (timer/software)
-    interrupts as external interrupts are usually generated by devices that
-    might require low interrupt service times.
-
-    Software interrupts are handled before internal timer interrupts,
-    because internal timer interrupts are usually intended for time slicing,
-    where time precision is less important, whereas software interrupts are
-    used for inter-processor messaging. Software interrupts can be avoided
-    when high-precision timing is required, or high-precision timer
-    interrupts can be routed via a different interrupt path. Software
-    interrupts are located in the lowest four bits of `mip` as these are
-    often written by software, and this position allows the use of a single
-    CSR instruction with a five-bit immediate.
-    ====
-
-    Restricted views of the `mip` and `mie` registers appear as the `sip`
-    and `sie` registers for supervisor level. If an interrupt is delegated
-    to S-mode by setting a bit in the `mideleg` register, it becomes visible
-    in the `sip` register and is maskable using the `sie` register.
-    Otherwise, the corresponding bits in `sip` and `sie` are read-only zero.
+  definedBy: Sm
+  description: "$copy: mip.yaml#/mip/description"
```

Comment:
why doesn't `$ref` work here? I think the intent is that it should, like so:

```yaml
description:
  $ref: "mip.yaml#/mip/description"
```

Further, we don't want $copy to be hidden in a string. Assuming we need to keep `$copy` for some reason, it should look like:

```yaml
description:
  $copy: "mip.yaml#/mip/description"
```

Reply:
I would have used $ref if it worked (it doesn't). That's because changing
   description: "foo"
to
   description:
      some-key: "foo"
would be changing "description" from a string to a hash and that would break everything using it (which it does).

Reply:
we really don't want "$copy: blah" in a string, so at the very least that needs to change. The open question is whether or not we can just use $ref here; to me, it seems like we should.